config/manifests/bases/nfd-operator.clusterserviceversion.yaml

```diff
@@ -571,11 +571,11 @@ spec:
   - discovery
   links:
   - name: Documentation operator
-    url: https://kubernetes-sigs.github.io/node-feature-discovery-operator/v0.4
+    url: https://kubernetes-sigs.github.io/node-feature-discovery-operator/v0.6
```

Comment:
For the master branch should this be `master`. No need to update anymore then (only on release branches)